torchci/lib/bot/ciflowPushTrigger.ts

```diff
@@ -129,38 +129,25 @@ async function handleLabelEvent(context: Context<"pull_request.labeled">) {
   }
 
   const label = context.payload.label.name;
-  if (label === "ci/master") {
-    let body = " `ci/master` label does not do anything. Did you mean `ciflow/trunk`?";
-    await context.octokit.issues.createComment(
-      context.repo({
-        body,
-        issue_number: context.payload.pull_request.number,
-      })
-    );
-    return;
-  }
   if (!isCIFlowLabel(label)) {
     return;
   }
-  const isOldCIFlowLabel =
-    label !== "ciflow/nightly" &&
-    label !== "ciflow/all" &&
-    label !== "ciflow/trunk" &&
-    label !== "ciflow/periodic" &&
-    !label.startsWith("ciflow/binaries");
-
-  if (isOldCIFlowLabel) {
+
+  // collected from .github/workflows/*
+  const valid_labels = ["ciflow/trunk", "ciflow/periodic", "ciflow/all", "ciflow/android", "ciflow/binaries",
```

Comment:
We should probably add a special case in for `ci/binaries` since I believe Pytorch Enterprise developers still use that particular label.